index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>apple-clone</title>
+    <link rel="stylesheet" href="styles/style.css">
+    <link href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">
+</head>
+<body>
+    <main>
+        <div class = "main-image">
+            <nav>
+                <ul>
+                    <li>
+                        <button class="apple-icon">
+                            <i class="fa fa-apple" aria-hidden="true"></i>
+                        </button>
+                    </li>
+                    <li>
+                        <button>Store</button>
+                    </li>
+                    <li>
+                        <button>Mac</button>
+                    </li>
+                    <li>
+                        <button>iPod</button>
+                    </li>
+                    <li>
+                        <button>iPhone</button>
+                    </li>
+                    <li>
+                        <button>iPad</button>
+                    </li>
+                    <li>
+                        <button>iTunes</button>
+                    </li>
+                    <li>
+                        <button>Support</button>
+                    </li>
+                    <li class="right button">
+                        <i class="fa fa-search custom-user"  aria-hidden="true"></i><input type="search" name="search">
+                    </li>
+                </ul>
+            </nav>
+            <div>
+                <h1>What will your verse be?</h1>
+                <p class="sub-heading">Everyone has something to share.See how people are using
+                    <br>iPad in their unique ways to add to the world's story
+                    <br>
+                    Watch the "Your verse" film and explore the stories
+                    <i class="fa fa-play-circle-o" aria-hidden="true"></i>
+                </p>
+            </div>
+        </div>
+        <aside>
+            <img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_iphone5s_2x.jpg"  alt="iPhone 5s. Forward thinking.">
+            <img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_iphone5c_2x.jpg" alt="iPhone 5c. For the colorful.">
+            <img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_30_years_2x.jpg"  alt="Happy Birthday, Mac. Explore 30 years of innovation on the Mac.">
+            <img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_supplier_responsibility_2x.jpg" alt="Supplier Responsibility The latest results of our worldwide efforts.">
+        </aside>
```

Comment:
- [ ] This section is aligned to the left side of the page, but it should be centered and aligned with the main background image.
![Screenshot from 2020-03-16 17-20-26](https://user-images.githubusercontent.com/51926937/76807594-7ac42e80-67aa-11ea-9e6d-eec45971b469.png)

You might not be able to see this due to your smaller screen, that's why the importance of giving the body a width (less than 100%)